docs/policy-static-pools.md

```diff
@@ -45,7 +45,7 @@ the original CMK. It tries to read the configuration from a location specified
 with `-static-pools-conf-dir` (`/etc/cmk` by default).
 Alternatively, you can provide a configuration file (YAML) by using the
 `-static-pools-conf-file` flag.
-See the [example config](../sample-configs/static-pools-policy.conf.example) for a
+See the [example config](/sample-configs/static-pools-policy.conf.example) for a
```

Comment:
It would have been nicer to distill this (and other similar fixes to the raw docs) into a separate 'docs: documentation fixes.'-commit. But it's fine like this now, no need to split it out any more.